apicast/src/oauth/keycloak.lua

```diff
@@ -237,9 +237,7 @@ function _M:authorize(service, client)
   end
 
   local url = resty_url.join(self.config.authorize_url, ngx.var.is_args, ngx.var.args)
-  local res = http_client.get(url)
-
-  _M.respond_and_exit(res.status, res.body, res.headers)
+  ngx.redirect(url)
```

Comment:
Hm, I think the RFC doesn't say anything about whether there can be a redirect.
Also, it refers to how Authorization Server should work, but not specifying how the proxy in front of it should behave.
To be honest, I am not sure what should be the way to go. On one hand, it probably is good to have it transparent (i.e. no redirects), but on the other hand, the user probably should know where exactly he is logging in (i.e. see RH SSO url in the address bar, so - redirect).